ssh/tailssh/incubator.go

```diff
@@ -222,13 +222,23 @@ func beIncubator(args []string) error {
 		}
 	}
 
+	homeDir := "/"
+	if userMeta, err := userLookupGetent(ia.localUser); err == nil {
```

Comment:
This file is cross-platform (see the line near the top: `//go:build linux || (darwin && !ios) || freebsd || openbsd`) but this userLookupGetEnt should only be used on Linux.

Reply:
Got it. I will refactor.